whotracksme/data/assets/trackerdb.sql

```diff
@@ -555,6 +555,24 @@ INSERT INTO "companies" VALUES('samba_tv','Samba TV',NULL,'https://samba.tv/lega
 INSERT INTO "companies" VALUES('sape','Sape Sapient Solution',NULL,NULL,'https://www.sape.ru/en',NULL);
 INSERT INTO "companies" VALUES('schibsted_asa','Schibsted ASA',NULL,NULL,'http://www.schibsted.com/',NULL);
 INSERT INTO "companies" VALUES('sentifi','Sentifi AG',NULL,'https://sentifi.com/page/privacy/','https://sentifi.com/',NULL);
+INSERT INTO "companies" VALUES('orange_france','Orange France',NULL,'http://c.orange.fr/donnees-personnelles.html','https://www.orange.fr/',NULL);
+INSERT INTO "companies" VALUES('dailymotion','Dailymotion',NULL,'http://www.dailymotion.com/legal/privacy','http://www.dailymotion.com/',NULL);
+INSERT INTO "companies" VALUES('le_monde.fr','Le Monde.fr
+',NULL,'http://www.lemonde.fr/service/donnees_personnelles.html','http://www.lemonde.fr/',NULL);
+INSERT INTO "companies" VALUES('accengage','Accengage',NULL,'https://www.accengage.com/privacy-policy/','',NULL);
+INSERT INTO "companies" VALUES('polar_inc','Polar Inc.',NULL,NULL,'https://polar.me/',NULL);
+INSERT INTO "companies" VALUES('nekudo','Nekudo',NULL,NULL,'https://nekudo.com/',NULL);
+INSERT INTO "companies" VALUES('cloudflare','Cloudflare',NULL,'https://www.cloudflare.com/security-policy/','https://www.cloudflare.com/',NULL);
+INSERT INTO "companies" VALUES('allegro.pl','Allegro',NULL,'https://allegro.pl/regulamin/pl#artykul-13.-prywatnosc-i-poufnosc','https://allegro.pl/','');
+INSERT INTO "companies" VALUES('pushwoosh','Pushwoosh',NULL,'https://www.pushwoosh.com/privacy-policy/','https://www.pushwoosh.com/',NULL);
+INSERT INTO "companies" VALUES('instart_logic_inc','Instart Logic Inc.',NULL,'https://www.instartlogic.com/company/legal','https://www.instartlogic.com/',NULL);
+INSERT INTO "companies" VALUES('the_guardian','The Guardian',NULL,'https://www.theguardian.com/info/privacy','https://www.theguardian.com/',NULL);
+INSERT INTO "companies" VALUES('optinmonster','OptinMonster',NULL,'https://optinmonster.com/privacy/','https://optinmonster.com/',NULL);
+INSERT INTO "companies" VALUES('onfocus','OnFocus',NULL,NULL,'http://onfocus.io/','5021');
+INSERT INTO "companies" VALUES('fastpic','FastPic',NULL,NULL,'http://fastpic.ru/',NULL);
+INSERT INTO "companies" VALUES('vk.com','Vk.com',NULL,'https://vk.com/privacy/eu','http://www.vk.com/','1034');
```

Comment:
vk.com is VKontakte, and is owned by Mail.RU group (https://www.crunchbase.com/organization/vk)

I see that Mail.RU was also acquired by Megafon, so that should be company entry, and all Mail.ru trackers should point to that.